packages/apps/tasks/src/App.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import {
+  ResetDialog,
+  GenericFallback,
+  ServiceWorkerToastContainer,
+  ThemeProvider,
+  appkitTranslations,
+} from "@dxos/react-appkit";
```

Comment:
Aren't we getting rid of `react-appkit`? These are straight outta the `bare` template. Can someone point me to a canonical app that's post-appkit.

cc: @richburdon @zhenyasav

Reply:
Also, we need to update the bare template. 🤔